src/test/java/org/testeditor/fixture/host/util/SingleLineReaderTest.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.host.util;
+
+import org.testeditor.fixture.host.Locator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SingleLineReaderTest {
+
+    @Test
+    public void readSingleLineTest() {
+        // given
+        List<String> lines = new ArrayList<String>();
+        lines.add("data:  01/01/2000 14:27:36        AMD MENUE FOR ABCDEFGHIJ          Panelid  - PAN1234");
+        lines.add("data:  Coroneradmin = TROLL                                         Terminal - TM12345");
+        lines.add("data:  Menukey     = KANONE      Printkey  = NONE                  Modell   - 3278-00");
+        lines.add("data:  Hitkey       = PF6         Printer   = LPT1234               System   - DONOTIN");
+        lines.add("data:  Cmdprefix    = $                                             Node     - ABCDEFG");
+        lines.add("data:                                                                                 ");
+        lines.add("data:                                                                                 ");
+        lines.add("data:     Session     PF-Key    Description                                           ");
+        lines.add("data:                                                                                 ");
+        lines.add("data:   _ ABC         PF 1      ABC                                                   ");
+        lines.add("data:   _ DEF         PF 2      DEF                                                   ");
+        lines.add("data:   _ GHI         PF 3      GHI                                                   ");
+        lines.add("data:   _ JKL         PF 4      JKL                                                   ");
```

Comment:
The host screen appears to be copy&pasted between the tests. If those methods are put into a single class this could be put into a setup method of the classes test.